locales/templates/app.pot

```diff
@@ -191,6 +196,53 @@ msgid ""
 "<a href=\"%2$s\">Learn more</a></p>"
 msgstr ""
 
+#. Header for the Privacy and Security settings menu
+msgctxt "preference_privacy_and_security_header"
+msgid "Privacy & Security"
+msgstr ""
+
+#. Preference summary for settings related to privacy and security
+msgctxt "preference_privacy_and_security_summary"
+msgid "Tracking, cookies, data choices"
+msgstr ""
+
+#. Preference category for settings related to search
+msgctxt "preference_search_summary"
+msgid "Set default, autocomplete"
```

Comment:
Maybe it's me, but this string sounds a bit weird. Could you please just clarify a bit more in what context it appears exactly (if you have a screenshot that would be even better), because right now, I have a hard time understanding how the 2 parts of the sentence go together ("Set default" + "autocomplete")